> ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2.17.55.0, 2.17.55.0]]

--- a/manifests/a/Amazon/AWSCLI/2.17.56/Amazon.AWSCLI.installer.yaml
+++ b/manifests/a/Amazon/AWSCLI/2.17.56/Amazon.AWSCLI.installer.yaml
@@ -19,7 +19,7 @@ ProductCode: '{07A75BD9-B3F5-4DAF-AE3C-53FCA6087C91}'
 AppsAndFeaturesEntries:
 - DisplayName: AWS Command Line Interface v2
   Publisher: Amazon Web Services
-  DisplayVersion: 2.17.55.0
+  DisplayVersion: 2.17.56.0
   ProductCode: '{D4D5429A-98F9-4016-9F12-27F948BD8A36}'
   UpgradeCode: '{E1C1971C-384E-4D6D-8D02-F1AC48281CF8}'
 InstallationMetadata:

